test/README.md

```diff
@@ -62,31 +100,16 @@ There's couple of things you need to install before running e2e tests locally.
    go get -u k8s.io/test-infra/kubetest
```

Comment:
e2e-tests.sh will now install kubetest for you if it's not in the system. Can you please remove this item?

Reply:
Done.